### PR TITLE
CORDA-2511 Upgrade notes for C4 need to include required minimum previous Corda version

### PR DESCRIPTION
**Pull request by szymonsztuka**
The Corda 4.x  upgrade notes include guidance to upgrade from minimum Corda 3.2 version
(https://r3-cev.atlassian.net/browse/CORDA-2511).

### CODE REVIEW

File: docs/source/node-upgrade-notes.rst [hunk 1]
```diff
@@ -38,7 +38,15 @@ user guide to learn how to make backups.
 
 We provide some :ref:`backup recommendations <backup-recommendations>` if you'd like more detail.
 
-Step 3. Replace ``corda.jar`` with the new version
+Step 3. Upgrade a database to a minimum Corda 3.2 version
```
Comment (anthonykeenan) [inline docs/source/node-upgrade-notes.rst]: Upgrade the node database to Corda 3.2 or later

File: docs/source/node-upgrade-notes.rst [hunk 2]
```diff
@@ -38,7 +38,15 @@ user guide to learn how to make backups.
...
+---------------------------------------------------------
+
+Ensure you are upgrading from at least Corda 3.2 version.
```
Comment (anthonykeenan) [inline docs/source/node-upgrade-notes.rst]: Ensure your node is running Corda 3.2 or later.

File: docs/source/node-upgrade-notes.rst [hunk 3]
```diff
@@ -38,7 +38,15 @@ user guide to learn how to make backups.
...
+The release 3.2 required a database table name change and for PostgreSQL a column type change.
```
Comment (anthonykeenan) [inline docs/source/node-upgrade-notes.rst]: Corda 3.2 required a database table name change and column type change in PostgreSQL.

File: docs/source/node-upgrade-notes.rst [hunk 4]
```diff
@@ -38,7 +38,15 @@ user guide to learn how to make backups.
...
+The changes needed to applied to the database before upgrading to Corda 4.0.
```
Comment (anthonykeenan) [inline docs/source/node-upgrade-notes.rst]: These changes need to be applied to the database

File: docs/source/node-upgrade-notes.rst [hunk 5]
```diff
@@ -38,7 +38,15 @@ user guide to learn how to make backups.
...
+Refer to `Corda 3.2 release notes <https://docs.corda.net/releases/release-V3.4/upgrade-notes.html#v3-1-to-v3-2>`_.
```
Comment (anthonykeenan) [inline docs/source/node-upgrade-notes.rst]: for further information

**Comment by szymonsztuka** [comment]
@anthonykeenan all fixes have been added

File: docs/source/node-upgrade-notes.rst [hunk 6]
```diff
@@ -38,7 +38,16 @@ user guide to learn how to make backups.
 
 We provide some :ref:`backup recommendations <backup-recommendations>` if you'd like more detail.
 
-Step 3. Replace ``corda.jar`` with the new version
+Step 3. Upgrade the node database to Corda 3.2 or later
+-------------------------------------------------------
+
+Ensure you node is running Corda 3.2 or later.
```
Comment (anthonykeenan) [inline docs/source/node-upgrade-notes.rst]: your

**Pull request merged**